x-pack/plugins/upgrade_assistant/public/application/components/tabs/overview/deprecation_logging_toggle.tsx

```diff
@@ -4,18 +4,16 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-import axios from 'axios';
 import React from 'react';
 
 import { EuiLoadingSpinner, EuiSwitch } from '@elastic/eui';
 import { injectI18n } from '@kbn/i18n/react';
 
-import { HttpSetup } from 'src/core/public';
+import { HttpSetup } from 'kibana/public';
```

Comment:
I don't think there is a difference, but for some reason when I import existing things (not types) from there it doesn't seem to work 🤷‍♂ . Also I believe some of these changes were introduced by my IDE.

I will change them all to `src/core/public` so that it is consistent here at least.